apps/dashboard/src/components/subscribers/subscriber-activity-list.tsx

```diff
@@ -0,0 +1,174 @@
+import { motion } from 'motion/react';
+import { FaCode } from 'react-icons/fa6';
+import { IActivity, JobStatusEnum, WorkflowOriginEnum } from '@novu/shared';
+
+import { JOB_STATUS_CONFIG } from '@/components/activity/constants';
+import { ActivityEmptyState } from '@/components/activity/activity-empty-state';
+import { RouteFill } from '@/components/icons/route-fill';
+import TruncatedText from '@/components/truncated-text';
+import { StatusBadge, StatusBadgeIcon } from '@/components/primitives/status-badge';
+import { getActivityStatus } from '@/components/activity/helpers';
+import { TimeDisplayHoverCard } from '@/components/time-display-hover-card';
+import { formatDateSimple } from '@/utils/format-date';
+import { TooltipContent, TooltipTrigger } from '@/components/primitives/tooltip';
+import { Tooltip } from '@/components/primitives/tooltip';
+import { cn } from '@/utils/ui';
+import { Skeleton } from '@/components/primitives/skeleton';
+
+const statusToToltipStyles: Record<string, string> = {
+  completed: 'before:bg-success-lighter before:border before:border-success-lighter text-success-base',
+  pending: 'before:bg-warning-lighter before:border before:border-warning-lighter text-warning-base',
+  failed: 'before:bg-error-lighter before:border before:border-error-lighter text-error-base',
+  disabled: 'before:bg-faded-lighter before:border before:border-faded-light text-faded-base',
+};
+
+const staggerSettings = (index: number) => ({
+  initial: { opacity: 0, y: 20 },
+  animate: { opacity: 1, y: 0 },
+  exit: { opacity: 0, y: -20 },
+  transition: {
+    duration: 0.4,
+    delay: index * 0.01,
+    ease: [0.25, 0.1, 0.25, 1],
+  },
+});
+
+export const SubscriberActivityList = ({
```

Comment:
this is the subscriber activity list component, it handles loading state, no result, and no result for filters